index.js

```diff
@@ -31,9 +31,10 @@ function sseMiddleware(__, res, next) {
     if (id) {
       res.write(`id: ${id}\n`);
     }
-
+    if (evt) {
```

Comment:
Okay, probably it wouldn't be easy to determine the order of arguments if all of them are strings.. Let me think please